src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,98 @@
-export const Pagination = () => {};
+import { useCallback } from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  total: string[],
+  currentPage: number,
+  perPage: number,
+  onPageChange: (value: number) => void | number,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total, currentPage, perPage, onPageChange,
+}) => {
+  const end = Math.ceil(total.length / perPage);
+
+  const displayPagination = useCallback((units: string[], currPage: number) => {
+    const pages = [];
+
+    // eslint-disable-next-line no-plusplus
+    for (let i = 1; i <= end; i++) {
+      pages.push(
+        <li
+          key={units[i]}
```

Comment:
Replace `key={units[i]}` with `key={i}` to avoid duplicate keys in case `units` has duplicate values.